src/main/java/com/example/bank/service/demo/service/HttpClient.java

```diff
@@ -0,0 +1,27 @@
+package com.example.bank.service.demo.service;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.math.BigDecimal;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HttpClient {
+    public BigDecimal sendHttpRequest(String path) {
```

Comment:
It's better to return result from this method so it will be more reusable or change the method name. It's not predictable that method sendHttpRequest returns BigDecimal